lib/rails/generators/preview/templates/component_preview.rb.tt

```diff
@@ -0,0 +1,11 @@
+require "test_helper"
+
+class <%= class_name %>ComponentPreview < ViewComponent::Preview
+  def with_useful_content
+  <%- if initialize_signature -%>
+    render(<%= class_name %>Component.new(<%= initialize_signature %>))
+  <%- else -%>
+    render(<%= class_name %>Component.new)
+  <%- end -%>
```

Comment:
I'm cautious of this approach, as ideally we'd couple the arguments passed to the component to the parameters accepted by its initializer. Perhaps we could add this functionality in a follow-up PR?

In the meantime it might be easier to simply add this as comment:

```suggestion
  # render(<%= class_name %>.new)
```

Reply:
Ok, may be it's not a correct approach. I'll let someone more experienced deal with this later.